client/src/styles/VisibilityOptions.scss

```
@@ -0,0 +1,3 @@
+.VisibilityOptions.field .visibility-options__option {
```

```suggestion
.visibility-options.field .visibility-options__option {
```
My suggestion assumes that you have added the `.field` for specificity - if not can we get away with just `.visibility-options__option`?

---

Yeah it's for specificity unfortunately.
I'll double check if it's needed. I presume the dist file for this module will always load after the class it's overriding.

---

... I've forgotten to lower case a class anyway, I'll check and fix both.